rootfs/scheduler/__init__.py

```diff
@@ -961,7 +968,7 @@ def _create_rc(self, namespace, name, image, command, **kwargs):  # noqa
 
             time.sleep(1)
 
-        return resp.json()
+        return response
```

Comment:
is there a particular reason why we don't want to return the json object here? Miind making this a separate commit with justification, at least?